src/app/components/cells/text/SelectableShortText.jsx

```diff
@@ -31,33 +39,64 @@ const enhance = compose(
   needsAPIData
 );
 
+function useOutsideAlerter(callback, ref) {
+  useEffect(() => {
+    function handleClickOutside(event) {
+      if (ref.current && !ref.current.contains(event.target)) {
+        callback();
+      }
+    }
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [ref]);
+}
+
 const SelectableShortText = props => {
   const {
     focusTable,
     onChange,
     onFinish,
     requestedData,
     setCellKeyboardShortcuts,
-    value
+    value,
+    column,
+    actions: { setPreventCellDeselection }
   } = props;
-
+  const shorttextRef = useRef(null);
   const [completions, setCompletions] = useState([]);
   const [isCompletionSelected, setIsCompletionSelected] = useState(false);
   const [selectedIdx, setSelectedIdx] = useState(0);
   const [shouldInvertList, setShouldInvertList] = useState(false);
+  const [shouldShowErrorState, setShouldShowErrorState] = useState(false);
+  const errorCssClass =
+    shouldShowErrorState && isTextTooShort(column, value)
+      ? "selectable-shorttext_error"
+      : "";
 
   useEffect(() => setCellKeyboardShortcuts({}));
+  useOutsideAlerter(() => {
+    setShouldShowErrorState(true);
+  }, shorttextRef);
 
   const handleTextChange = event => {
     const inputValue = event.currentTarget.value;
     const completionsForValue = extractAndFilterCompletions(
       inputValue,
       requestedData
     );
+    if (inputValue.length > column.maxLength) return;
```

Comment:
[Qeustion] Verstehe ich es richtig, dass hier einfach abgebrochen wird und die Textänderung verworfen wird?

Reply:
Ja

Reply:
Hm, das muss ich dann mal in der UI testen/sehen wie sich das verhält. Klingt erstmal blöd, wenn plötzlich alles weg ist. Der User sollte immer die Möglichkeit haben es zu korrigieren und wenn dann nur aktiv "abbrechen" müssen. Ggf. mit Fehlermeldung.

Reply:
Ok, wenn es so ist in der UI dann passt es. Ich hatte gerade erwartet, dass man mehr schreiben kann und beim speichert ist es dann einfach weg. Deswegen meinte ich ja, das muss ich in der UI sehen wie es sich verhält :)
Ist dann erstmal resolved.